up42/auth.py

```diff
@@ -144,10 +171,39 @@ def _endpoint(self) -> str:
         return f"https://api.up42.{self.env}"
 
     def _get_token(self):
+        if self.auth_type == AuthType.PROJECT.value:
+            self._get_token_project_based()
+        elif self.auth_type == AuthType.ACCOUNT.value:
+            self._get_token_account_based()
+
+    def _get_token_account_based(self):
+        """Account based authentication via username and password."""
+        try:
+            req_headers = {
+                "Content-Type": "application/x-www-form-urlencoded",
+            }
+            req_body = {
+                "grant_type": "password",
+                "username": self.credentials_id,
+                "password": self.credentials_key,
+            }
+            token_response = requests.post(
+                url=self._endpoint() + "/oauth/token",
+                data=req_body,
+                headers=req_headers,
+                timeout=1200,
```

Comment:
20 mins seem like a very long time to wait before we raise Timeout exception. Consider lowering it something like 120 seconds.

Also, consider defining this as a global variable and passing it here. Something like:
`TIMEOUT_DURATION = 120`
`timeout = TIMEOUT_DURATION`